README.md

```diff
@@ -231,28 +245,32 @@ async def get_year_resource(self, link: str, year: int) -> ResourceInfo:
 
   return ResourceInfo(local_path=download_path, partitions={"year": year})
 ```
+
 This method should handle the following steps:
-* identify the specific download link for the file(s) in the partition
-* rename the file to match our data conventions. We rename files to match the format
+
+- identify the specific download link for the file(s) in the partition
+- rename the file to match our data conventions. We rename files to match the format
 `datasource-partition.ext` - e.g. `eia860-1990.zip`.
-* construct the path to where we want to temporarily store the file locally, using
+- construct the path to where we want to temporarily store the file locally, using
```

Comment:
were all of these swaps from `*` to `-` necessary?

Reply:
It was rendering fine on GitHub but my markdown linter was complaining about them and some other minor formatting details, and I just accepted its suggestions.